FastTimeSeries/src/tasks/classification.jl

```diff
@@ -26,4 +26,33 @@ _tasks["tsclfsingle"] = (
         class label associated with it.
         """,
     package = @__MODULE__,
-)
\ No newline at end of file
+)
+
+# ## Tests
+
+@testset "TabularClassificationSingle [task]" begin
+    data, blocks = load(datarecipes()["atrial"])
+
+    task = TSClassificationSingle(blocks, data)
+    FastAI.testencoding(task.encodings, task.blocks.sample)
+    FastAI.checktask_core(task)
+    @test_nowarn tasklossfn(task)
+    
+    @testset "`encodeinput`" begin 
+        row = FastAI.mockblock(task.blocks[1])
+
+        xtrain = FastAI.encodeinput(task, Training(), row);
+        @test length(xtrain[1,:]) == task.blocks.input.obslength
+        @test length(xtrain[:,1]) == task.blocks.input.nfeatures
+
+        @test eltype(xtrain[1,:]) <: Number
```

Comment:
Is the input expected to be non-contiguous here? If not, I would compare against `size` instead of lengths of slices.